### PR TITLE
strings tasks are completed

### PR DESCRIPTION
**Pull request by emakhachek**

### CODE REVIEW

File: Exercises/strings/strings.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// Task #4
+console.log(`My favorite animal is ${c}`);
+
+// Task #5
+function CapitalizeFirstLetters(text) {
+    let temp = '';
+    let res = '';
+    for (let i = 0; i < text.length; i++) {
+        temp += text[i];
+        if (text[i] == ' ' || i == text.length - 1) {
```
Comment (lizarusi) [inline Exercises/strings/strings.js:21]: wow

Comment (lizarusi) [reply Exercises/strings/strings.js:21]: you can simplify using `split` but gj

File: Exercises/strings/strings.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const c = 'Cat';
+
+// Task #4
+console.log(`My favorite animal is ${c}`);
+
+// Task #5
+function CapitalizeFirstLetters(text) {
+    let temp = '';
+    let res = '';
+    for (let i = 0; i < text.length; i++) {
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: i += 1

**Review by lizarusi — CHANGES_REQUESTED** [review]

File: Exercises/strings/strings.js (hunk 1 above)
Comment (Mike-Savin) [reply Exercises/strings/strings.js:21]: 👍

File: Exercises/strings/strings.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// Task #1
+const a = "\u2764";
+console.log(a);
+
+// Task #2
+const b = "This book costs 100\$";
```
Comment (Mike-Savin) [inline Exercises/strings/strings.js]: $ is not a symbol that should be escaped

**Review by Mike-Savin — CHANGES_REQUESTED** [review]

File: Exercises/strings/strings.js (hunk 1 above)
Comment (Mike-Savin) [reply Exercises/strings/strings.js:21]: btw function names should not start with capital letters

File: Exercises/strings/strings.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+console.log(b);
+
+// Task #3
+const c = 'Cat';
+
+// Task #4
+console.log(`My favorite animal is ${c}`);
+
+// Task #5
+function CapitalizeFirstLetters(text) {
```
Comment (Mike-Savin) [inline Exercises/strings/strings.js]: function name should be in lowerCamelCase

**Review by Mike-Savin — CHANGES_REQUESTED** [review]

**Review by Mike-Savin — APPROVED** [review]

**Pull request merged**